sdk/monitor/monitor-query/src/models/publicMetricsModels.ts

```diff
@@ -62,6 +59,14 @@ export interface Metric {
   timeseries: TimeSeriesElement[];
 }
 
+/** Represents a metric metadata value. */
+export interface MetadataValue {
```

Comment:
Should this be a QueryMetadataValue or LogQueryMetadataValue? It's a little long but `MetadataValue` is very generic

Reply:
Good spot!

Interestingly in the other SDKs they've just turned this "array of (key,value)" into a Dictionary, which will eliminate this type altogether....I'm going to do that real quick.

Reply:
Oof, okay, I'm going to leave this one in place. There are a few interesting variations here, and I think I need a better test before I change it. Slated for next release.